src/components/common/buttons/TaskButton.jsx

```diff
@@ -29,4 +30,35 @@ const TaskButton = ({ buttonText, url, eventTag }) => {
   )
 }
 
+export const WalletV2Continue = ({ buttonText, dontShowAgain, onDismiss }) => {
+  const goToWalletV2 = async () => {
+    try {
+      if (dontShowAgain) {
+        fireEvent('migration_denied')
```

Comment:
we have a standard format for events using constants

Reply:
changed to const format